### DISCUSSION TITLE
New to coding. How does this extension work?

### DISCUSSION THREAD

**Discussion by KodaLover** [htmlhint/HTMLHint#666] [Q&A] [answered]
I am very new to coding, and am not sure why htmlhint is not working for me in VSCode. I've installed W3C Web Validator
 and it shows many errors with my coding, but HTMLhint doesn't show any errors, and in fact, doesn't even appear to work. Do you have any tips? I've tried to find an easy tutorial on the basics of using it, but have not been able to find anything.

**Comment by coliff** [comment, marked as answer]
Sorry for the delay replying! I'd recommend using the VS Code Extension:

https://marketplace.visualstudio.com/items?itemName=HTMLHint.vscode-htmlhint
